packages/oncokb-frontend-commons/src/components/StatusHelpers.tsx

```diff
@@ -0,0 +1,34 @@
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
```

Comment:
This is the duplicate of the one in `react-mutation-mapper`. We should probably move `StatusHelpers.tsx` into `cbioportal-frontend-commons` to be able to reuse it in both packages.

@zhx828 @scarrero4660 @alisman Note that `DefaultTooltip` component has a `mobx` dependency, so I think when we import components from `cbioportal-frontend-commons` we end up with `mobx` dependency.

Reply:
@onursumer yeah, seems like the biggest concern of this PR. Particularly two imports, `DefaultTooltip` and `ICache/SimpleCache`
@scarrero4660 I saw you moved SimpleCache from RMM to oncokb commons, I don't think this is correct.